masappcli/masappcli.py

```diff
@@ -446,9 +470,7 @@ def upload_and_analyse_app(self, app_path, package_name_origin=None, workgroup=N
 
         """
 
-        if workgroup == None:
-            self.store_workgroup(0)
-        else:
+        if workgroup != None:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [comparison to None should be 'if cond is not None:'](https://app.codacy.com/app/alopezna5/mASAPP_CI/pullRequest?prid=4655639)